dynamiqs/utils/jax_utils.py

```diff
@@ -132,3 +140,15 @@ def set_precision(precision: Literal['simple', 'double']):
             f"Argument `x` should be a string 'simple' or 'double', but is"
             f" '{precision}'."
         )
+
+    if matmul_precision == 'low':
+        jax.config.update('jax_default_matmul_precision', 'bfloat16')
+    elif matmul_precision == 'high':
+        jax.config.update('jax_default_matmul_precision', 'bfloat16_3x')
+    elif matmul_precision == 'highest':
+        jax.config.update('jax_default_matmul_precision', 'float32')
```

Comment:
It seems strange that we cannot go to float64 for matrix product no? What is the point of f64 then?

Reply:
We can, it's just that this API is very poorly written from Jax. Float32 actually means float32 or float64 depending on the global precision. Figure out! 🤷

Reply:
We should comment that somewhere

Reply:
See https://jax.readthedocs.io/en/latest/jax.lax.html#jax.lax.Precision, I've switched the key to `'highest'` which is not ambiguous.